composer/utils/file_helpers.py

```diff
@@ -339,6 +341,64 @@ def get_file(
     Raises:
         FileNotFoundError: If the ``path`` does not exist.
     """
+    if path.endswith('.symlink'):
+        with tempfile.TemporaryDirectory() as tmpdir:
+            symlink_file_name = os.path.join(tmpdir, 'file.symlink')
+            # Retrieve the symlink
+            _get_file(
+                path=path,
+                destination=symlink_file_name,
+                object_store=object_store,
+                overwrite=False,
+                progress_bar=progress_bar,
+            )
+            # Read object name in the symlink
+            with open(symlink_file_name, 'r') as f:
+                real_path = f.read()
+
+        # Recurse
+        return get_file(
+            path=real_path,
+            destination=destination,
+            object_store=object_store,
+            overwrite=overwrite,
+            progress_bar=progress_bar,
+        )
+
+    try:
+        _get_file(
+            path=path,
+            destination=destination,
+            object_store=object_store,
+            overwrite=overwrite,
+            progress_bar=progress_bar,
+        )
+    except FileNotFoundError as e:
+        if path.endswith('.symlink'):
```

Comment:
nit: if I am reading your code correctly, this condition will always evaluate to `False`, because of the check above.

otherwise, lgtm

Reply:
Thanks, good catch. I ended up refactoring the logic and forgot to remove this now-redundant check

Reply:
Fixed